### PR TITLE
Improve the UX of the clip-plane and wireframe buttons

### LINKED ISSUE #406: Improve the UX of the clip-plane and wireframe buttons
## Problems

Today, the clip-plane toolbar button shows a dialog to select two options:
- selected
- show Clip Plane

In the other side, the wireframe button does not show a dialog and directly apply the wireframe effect. Though once toggled, there is no visual indication in the toolbar that the button has been toggled.

Also, refreshing the page or reopening the file does not keep the clip-plane or wireframe states. We could fix this by saving the states in the schema.

## Suggestions

- We could make the behavior more homogeneous by making the clip-plane toolbar button not showing a dialog and directly toggling the clip-plane effect (we would always show the clip-plane in the 3D scene by default).
- We should show a visual indication in the toolbar that toggle buttons are toggled
- We should save the states of the clip-plane and wireframe effect in the schema

### PR DESCRIPTION
**Pull request by arjxn-py**
https://github.com/user-attachments/assets/42d2a6d4-728a-49d0-a914-c45b449bffe4

Should close #406

### CODE REVIEW

**Comment by arjxn-py** [comment]
Saving these properties in schema to preserve state remains.

File: packages/base/src/commands.ts [hunk 1]
```diff
@@ -971,6 +937,17 @@ export function addCommands(
         return;
       }
       current.wireframe = !current.wireframe;
+
+      const buttonNode = document.querySelector(
+        `[data-command="${CommandIDs.wireframe}"]`
+      );
```
Comment (trungleduc) [inline packages/base/src/commands.ts:943]: If I have multiple CAD documents opened. This selector will not select the correct one.

File: packages/base/src/commands.ts [hunk 2]
```diff
@@ -1079,15 +1056,30 @@ export function addCommands(
...
+
+      const { enabled, showClipPlane } = panel.clipView;
+      panel.clipView = {
+        enabled: enabled,
+        showClipPlane: showClipPlane
+      };
+
+      const buttonNode = document.querySelector(
+        `[data-command="${CommandIDs.updateClipView}"]`
+      );
```
Comment (trungleduc) [inline packages/base/src/commands.ts:1075]: Same here

File: packages/base/src/commands.ts (hunk 1 above)
Comment (arjxn-py) [reply packages/base/src/commands.ts:943]: Ahh yes, you're right

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**